### ISSUE TITLE
What is the expected format of dataloaders generated from TimeSeriesDataset?

### ISSUE THREAD

**Issue by dempseyryan** [jdb78/pytorch-forecasting#1274]
- PyTorch-Forecasting version: 0.10.3
- PyTorch version: 1.13.1
- Python version: 3.9.16
- Operating System: Google Collab Linux kernel

### Expected behavior

I have 5 targets. When I load from the test set dataloader, this is what I get:

```
# this returns a list of len 2, where each element is a list of length 5
# since I have 5 targets, I assume the outer list is batches, and the inner lists are for targets
actuals = [y[0] for x, y in iter(test_dl)]
```

```
actuals[0][0].shape
torch.Size([128, 9])

actuals[1][0].shape
torch.Size([128, 9])
```
I assume this is group, time_idx. Therefore, I try to do this, in order to concatenate the actuals:

```
actuals = torch.cat([torch.stack(val) for val in actuals], dim=1).numpy()
```

I am expecting this to have shape (5, 128+22, 9), which it has: (5, 150, 9)

But the values seem to be in the wrong order. I know that in my ground truth, none of the targets are 1.0 at the same time, however, this is the result of actuals:

### Actual behavior

```
array([[[1., 1., 1., ..., 1., 1., 1.],
        [1., 1., 1., ..., 1., 1., 1.],
        [1., 1., 1., ..., 1., 1., 1.],
        ...,
        [0., 0., 0., ..., 0., 0., 0.],
        [0., 0., 0., ..., 0., 0., 0.],
        [0., 0., 0., ..., 0., 0., 0.]],

       [[1., 1., 1., ..., 1., 1., 1.],
        [1., 1., 1., ..., 1., 1., 1.],
        [1., 1., 1., ..., 1., 1., 1.],
        ...,
        [0., 0., 0., ..., 0., 0., 0.],
        [0., 0., 0., ..., 0., 0., 0.],
        [0., 0., 0., ..., 0., 0., 0.]],

       [[1., 1., 1., ..., 1., 1., 1.],
        [1., 1., 1., ..., 1., 1., 1.],
        [1., 1., 1., ..., 1., 1., 1.],
        ...,
        [0., 0., 0., ..., 0., 0., 0.],
        [0., 0., 0., ..., 0., 0., 0.],
        [0., 0., 0., ..., 0., 0., 0.]],

       [[1., 1., 1., ..., 1., 1., 1.],
        [1., 1., 1., ..., 1., 1., 1.],
        [1., 1., 1., ..., 1., 1., 1.],
        ...,
        [0., 0., 0., ..., 0., 0., 0.],
        [0., 0., 0., ..., 0., 0., 0.],
        [0., 0., 0., ..., 0., 0., 0.]],

       [[1., 1., 1., ..., 1., 1., 1.],
        [1., 1., 1., ..., 1., 1., 1.],
        [1., 1., 1., ..., 1., 1., 1.],
        ...,
        [0., 0., 0., ..., 0., 0., 0.],
        [0., 0., 0., ..., 0., 0., 0.],
        [0., 0., 0., ..., 0., 0., 0.]]], dtype=float32)
```
All of the targets are either 0 or 1 together, which means they must be from separate series, meaning I've concatenated them incorrectly. What is the format of the dataloader when using TimeSeriesDataSet? I would like to analyze all the targets in the same time_idx and group.
### DataSet code

```
  training = TimeSeriesDataSet(
    data[lambda x: x.frame <= training_cutoff],
    time_idx="frame",
    target=["thumb", "index", "middle", "ring", "pinkie"],
    # target=["finger"],
    group_ids=["subject_id", "trial", "hand"],
    min_encoder_length=1,
    max_encoder_length=max_encoder_length,
    min_prediction_length=1,
    # min_prediction_length=max_prediction_length,
    max_prediction_length=max_prediction_length,
    # static_reals=["lateral", "depth"],
    # time_varying_known_reals=["frame"],
    time_varying_known_reals=list_of_mav_columns(),
    # time_varying_unknown_reals=list_of_mav_columns() + ["thumb", "index", "middle", "ring", "pinkie"],
    time_varying_unknown_reals=["thumb", "index", "middle", "ring", "pinkie"],
    # time_varying_unknown_categoricals=["finger"],
    # time_varying_unknown_categoricals=["thumb", "index", "middle", "ring", "pinkie"],
    add_relative_time_idx=True,
    add_target_scales=True,
    add_encoder_length=True,
    allow_missing_timesteps=True,
    target_normalizer=MultiNormalizer([TorchNormalizer(method='identity', center=False, transformation=None)]*5),
    randomize_length=False
  )

  # create test set (predict=True) which means to predict the last max_prediction_length points in time
  # for each series
  test = TimeSeriesDataSet.from_dataset(training, data, predict=True, min_prediction_idx=training_cutoff+1, stop_randomization=True)

  # create dataloaders for model
  train_dl = training.to_dataloader(train=True, batch_size=batch_size, num_workers=32)
  test_dl = test.to_dataloader(train=False, batch_size=batch_size * 4, num_workers=32)
```
